beacon-chain/db/state.go

```diff
@@ -125,6 +126,11 @@ func (db *BeaconDB) HeadState(ctx context.Context) (*pb.BeaconState, error) {
 	return beaconState, err
 }
 
+// StateHash returns the hash of the current state from the db.
```

Comment:
is this for the state of the head?

if yes, we should use head in the name to be explicit

Reply:
Alright, sure thing. Changed it to HeadStateRoot to be more explicit